framework/error.go

```diff
@@ -17,3 +17,10 @@ var invalidConfigError = microerror.New("invalid config")
 func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
+
+var noResourceRouterError = microerror.New("no resource router")
```

Comment:
This should be `no resource set`

Reply:
Good catch buddy. See https://github.com/giantswarm/operatorkit/pull/165.